packages/modules/web_themes/standard/source/quasar.config.ts

```diff
@@ -17,6 +17,7 @@ export default configure((/* ctx */) => {
 
     // https://v2.quasar.dev/quasar-cli-vite/quasar-config-js#css
     css: ['app.scss'],
+    cssAddon: true,
```

Comment:
In der Doku habe ich keine genauen Angaben hierzu finden können. Warum ist der Eintrag notwendig?

Reply:
I thought it was needed for the CSS themes in quasar.variables.scss but it appears to work without it and can thus be deleted.